src/components/Home/HomePage.jsx

```diff
@@ -0,0 +1,302 @@
+import React from "react";
+import "../../styles/root.css";
+import { Link } from "react-router-dom";
+
+export function HomePage() {
+	return (
+		<>
+			<main className="col-12">
+				<div className="big-container">
+					<img
+						className="imageContainer"
+						src="../images/img2.webp"
+						width="900"
+						height="700"
+						alt=""
```

Comment:
try to give alt attribute some description about the image..

Reply:
For performance basis. You can add --- loading="lazy"------ to img tag. It will help in optimization .

Reply:
Okay, I'll do that.